colorize.lisp

```diff
@@ -283,9 +283,13 @@
             (mapcar color-formatter scan)
             (funcall (coloring-type-formatter-after-hook coloring-type-object)))))
 
+(defun encoder-fn (encoder)
+  (or (find-symbol (princ-to-string encoder) :html-encode)
+      (error "Unable to find encoder function ~A in the html-encode package" encoder)))
```

Comment:
Signalling the error here is easier to debug than if we allow NIL to be returned and fail later at ` (funcall encoder-fn token)`